<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostLiveClientClass</class>
 <widget class="QMainWindow" name="PostLiveClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PostLiveClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="FFmpegWidget" name="ffmpegWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>652</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_Devices">
     <property name="title">
      <string>&amp;Devices</string>
     </property>
    </widget>
    <addaction name="menu_Devices"/>
    <addaction name="actionFile"/>
   </widget>
   <widget class="QMenu" name="menu_Config">
    <property name="title">
     <string>&amp;Config</string>
    </property>
    <addaction name="action_Server"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Config"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Play">
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="action_Pause">
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="action123">
   <property name="text">
    <string>123</string>
   </property>
  </action>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="action_Server">
   <property name="text">
    <string>&amp;Server</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FFmpegWidget</class>
   <extends>QWidget</extends>
   <header>ffmpeg/FFmpegWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../PostLiveClient.qrc"/>
 </resources>
 <connections/>
</ui>
